src/components/Main.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import {movies} from '../data/movies';
+import Card from './Card';
+
+export const Main = () => {
+
+   movies?.forEach((x) => {
+      console.log(JSON.stringify(x));
+   })
+   
+
+  return (
+    <div className='layout_grid'>
+      {movies.map((x)=> {
+         return (<Card title={x.title} year={x.year} genders={x.genders} poster={x.poster}/>)
```

Comment:
`key` is missing.
Minor: when props are match one-to-one you can use spread operator.
```suggestion
         return (<Card {...x} />)
```